fitbenchmarking/controllers/base_controller.py

```diff
@@ -283,7 +283,8 @@ def cleanup(self):
 
         The flag corresponds to the following messages:
 
-        .. automethod:: fitbenchmarking.controllers.base_controller.Controller.flag()
+        .. automethod:: fitbenchmarking.controllers. \
```

Comment:
I think this has broken the docs -- [master](https://fitbenchmarking.readthedocs.io/en/latest/contributors/module_index/fitbenchmarking.controllers.base_controller.html#fitbenchmarking.controllers.base_controller.Controller.cleanup), [pr](https://fitbenchmarking.readthedocs.io/en/765_controllers-linting/contributors/module_index/fitbenchmarking.controllers.base_controller.html#fitbenchmarking.controllers.base_controller.Controller.cleanup)

Reply:
Good spot! I've now changed this back to being on one line so the docs are working, but am not sure how to avoid the flake8 line too long error?